stdlib/websub/src/main/ballerina/src/websub/natives.bal

```diff
@@ -50,10 +48,10 @@ function externStartUpHubService(handle basePath, handle subscriptionResourcePat
     class: "org.ballerinalang.net.websub.nativeimpl.HubNativeOperationHandler"
 } external;
 
-# Stop the Ballerina Hub, if started.
+# Stops the Ballerina Hub, if started.
 #
 # + hub - The `websub:Hub` object returned when starting the hub
-# + return - `()` if the Ballerina Hub had been started up and was stopped now, `error` if not
+# + return - `()` if the Ballerina Hub had been started up and was stopped now or else an `error` if not
```

Comment:
```suggestion
# + return - `()` if the Ballerina Hub had been started and was stopped now or else an `error` otherwise
```
Need to specify the error.